veg2hab/habitat.py

```diff
@@ -258,6 +258,27 @@ def try_to_determine_habkeuze(
 
             # ...of zijn er meerdere kloppende mitsen; Alle info van de kloppende mitsen meegeven
             if len(true_voorstellen) > 1:
+                if all(
+                    [
+                        voorstel.habtype == true_voorstellen[0].habtype
+                        for voorstel in true_voorstellen
+                    ]
+                ) and all(
+                    [
+                        voorstel.kwaliteit == true_voorstellen[0].kwaliteit
+                        for voorstel in true_voorstellen
+                    ]
+                ):
+                    return HabitatKeuze(
+                        status=KeuzeStatus.DUIDELIJK,
+                        habtype=true_voorstellen[0].habtype,
+                        kwaliteit=true_voorstellen[0].kwaliteit,
+                        habitatvoorstellen=true_voorstellen,
+                        opmerking=f"Er zijn meerdere habitatvoorstellen die aan hun mitsen/mozaieken voldoen; zie mits/mozk_opm voor meer info in format [opgegeven vegtype, potentieel habtype, mits/mozaiek]",
+                        mits_opmerking=f"Mitsen: {[[str(voorstel.onderbouwend_vegtype), voorstel.habtype, str(voorstel.mits), str(voorstel.mits.evaluation)] for voorstel in true_voorstellen]}",
```

Comment:
`str()` gaat automatisch in een fstring dacht ik. Of misschien wordt __format__ gebruikt, ook niet heel belangrijk.

Reply:
Als ik de strs weghaal krijg ik

`"Mitsen: [[SBB(klasse='50', verbond='a', associatie=None, subassociatie=None, derivaatgemeenschap=None, rompgemeenschap=None), 'H7220', NietGeautomatiseerdCriterium(toelichting='mits tufvorming plaatsvindt'), <MaybeBoolean.CANNOT_BE_AUTOMATED: 3>]]"`

In plaats van

`"Mitsen: [['50a', 'H7220', '(Niet geautomatiseerd: mits tufvorming plaatsvindt)', 'CANNOT_BE_AUTOMATED']]"`